src/seedu/addressbook/Main.java

```diff
@@ -84,6 +86,9 @@ private void runCommandLoopUntilExitCommand() {
         do {
             String userCommandText = ui.getUserCommand();
             command = new Parser().parseCommand(userCommandText);
+            if(userCommandText.equals("help")) {
```

Comment:
This one is very minor, just need to note that Java reserved words should be followed by a white space. eg. 
`if (userCommandText.equals("help")) {`